### PR TITLE
Adding support for articulated tongs

### PR DESCRIPTION
**Pull request by Onbusi**
Adding support in forge.lic for articulated/clockwork tongs.  Put use_tongs_as_shovel: false in base.yaml

### CODE REVIEW

**Comment by Sarvatt** [comment]
can you switch to spaces for indent instead of tabs please?

File: forge.lic [hunk 1]
```diff
@@ -293,19 +294,42 @@ class Forge
   end
 
   def add_fuel
-    stow_item('tongs')
```
Comment (rpherbig) [inline forge.lic]: I wonder if the code would be any clearer if you wrapped the `stow` and `get` here with an `if @articulated` check. Something like:

```
if @articulated
  # do new stuff
else
  # do old stuff
end

# push fuel like the old code

if @articulated
  # put away with new code
else
  # put away with old code
end
```

Comment (Onbusi) [reply forge.lic]: Alright, I'll work on doing that and testing it today

**Comment by rpherbig** [comment]
@Onbusi are you still working on this?

**Comment by BinuDR** [comment]
If this is still being worked on, could you include clockwork tongs too?

>tap clock tongs
You tap some clockwork-enhanced tongs with an attached shovel head that you are holding.
>adjust tong
You lock the tongs into a fully extended position.  Then you reach alongside the tong's body, and pull free the head of a shovel.
>tap tong
You tap some clockwork-enhanced tongs with an attached shovel head that you are holding.
>adjust tong
With a yank you fold the shovel head back alongside the tong's body.  Then you unlock the arms, readying them for use as tongs once more.

**Pull request closed without merging**